velox/functions/prestosql/aggregates/ReduceAgg.cpp

```diff
@@ -26,51 +26,20 @@ namespace {
 using ReduceAggAccumulator = functions::aggregate::SingleValueAccumulator;
 
 // clang-format off
-//  TODO: reduce_agg aggregate function doesn't lend itself well to vectorized
+//  reduce_agg aggregate function doesn't lend itself well to vectorized
 //  execution as it is effectively a data-dependent loop. It is hard to avoid
-//  evaluating lambda expressions on small number of rows at a time. This
-//  particular implementation doesn't try very hard and simply evaluates lambda
-//  expressions one row at a time. A relatively straightforward improvement
-//  could be similar to the algorithm implemented in the 'reduce' scalar lambda
-//  function.
+//  evaluating lambda expressions on small number of rows at a time. An original
+//  implementation didn't try very hard and simply evaluated lambda
+//  expressions one row at a time. This implementation is optimized to evaluate
```

Comment:
Do we still need this sentence?

> An original implementation didn't try very hard and simply evaluated lambda expressions one row at a time.

Reply:
I'd like to keep the description of the original implementation here for context.